src/mesh/GlobalData.hpp

```diff
@@ -0,0 +1,94 @@
+#pragma once
+
+#include <Eigen/Core>
+#include <stddef.h>
+#include <string>
+
+#include "logging/Logger.hpp"
+#include "precice/types.hpp"
+
+namespace precice {
+namespace mesh {
+class GlobalData;
+}
+} // namespace precice
+// TODO: GlobalData should not be inside mesh namespace
+
+// ----------------------------------------------------------- CLASS DEFINITION
+
+namespace precice {
+namespace mesh {
+/**
+ * @brief Describes a data value not belonging to a mesh.
+ */
+class GlobalData {
+public:
+  // @brief Possible types of data values.
+  //  enum DataType {
+  //    TYPE_UNDEFINED,
+  //    TYPE_DOUBLE,
+  //    TYPE_VECTOR
+  //  };
+
+  // @brief Name of an undefined data type.
+  //static const std::string TYPE_NAME_UNDEFINED;
+  // @brief Name of a double data type.
+  //static const std::string TYPE_NAME_DOUBLE;
+  // @brief Name of a vector data type.
+  //static const std::string TYPE_NAME_VECTOR;
```

Comment:
?

Reply:
Legacy copypasta from `Data.hpp`. Thanks for noticing!

Reply:
removed